xinference/model/llm/llm_family.py

```diff
@@ -782,6 +785,32 @@ def get_user_defined_llm_families():
         return UD_LLM_FAMILIES.copy()
 
 
+def match_model_size(
+    model_size: Union[int, str], spec_model_size: Union[int, str]
+) -> bool:
+    if model_size == spec_model_size:
+        return True
+
+    if isinstance(model_size, str):
+        try:
+            ms = int(model_size)
+            ss = int(spec_model_size)
```

Comment:
What if `model_size` is `"1_8"` and `spec_model_size` is `"18"`?
There's a misjudge here.

Reply:
ok, fixed it and add some test cases.